bn_deprecated.c

```diff
@@ -232,16 +232,20 @@ mp_err mp_n_root(const mp_int *a, mp_digit b, mp_int *c)
 #ifdef BN_MP_TORADIX_N_C
 mp_err mp_toradix_n(const mp_int *a, char *str, int radix, int maxlen)
 {
+   size_t length[1];
    if (maxlen < 0) {
       return MP_VAL;
    }
-   return mp_to_radix(a, str, (size_t)maxlen, radix);
+   *length = (size_t)maxlen;
+   return mp_to_radix(a, str, length, radix);
 }
 #endif
 #ifdef BN_MP_TORADIX_C
 mp_err mp_toradix(const mp_int *a, char *str, int radix)
 {
-   return mp_to_radix(a, str, SIZE_MAX, radix);
+   size_t length[1];
+   *length = SIZE_MAX;
+   return mp_to_radix(a, str, length, radix);
```

Comment:
size_t len = SIZE_MAX; ...&len) please

Reply:
No awkwardness then

Reply:
I really should use a bit less of that dreaded C&P, or what do you think? ;-)